libsolidity/interface/StandardCompiler.cpp

```diff
@@ -70,6 +74,36 @@ Json::Value formatFatalError(string const& _type, string const& _message)
 	return output;
 }
 
+Json::Value formatSourceLocation(SourceLocation const* location)
+{
+	Json::Value sourceLocation;
+	if (location && location->source && !location->source->name().empty())
```

Comment:
maybe reverse the condition to avoid the block & indent?

Reply:
Nevermind, I didn't see that you reuse the return

Reply:
though you could just do `return Json::Value()` I suppose..

Reply:
I would say the function is short enough...